### DISCUSSION TITLE
Connections - consider consolidating dropdown list of environments

### DISCUSSION THREAD

**Discussion by Cloud-Weaver** [swhitley/ERPHelper#30] [Ideas]
All of the **Impl** and **Sandbox** and **Sandbox Preview** URL's are the same for the majority of the tenants. 

It would be a good idea to consolidate them together (IMPL/SBX and PROD).  As for PREVIEW, the tenant alias (ex. "client__preview_" ) contains the flag Workday needs to direct the request correctly so there isn't a need for a user to select an endpoint for Preview. Currently, Studio provides the full list as seen now but I believe it's less overwhelming to users and reduces uncertainty when a request is having issues being processed by Workday.

 If that Preview flag is needed elsewhere in the plugin, then providing a checkbox to the user could be sufficient. Workday also uses a checkbox to control the preview flag in the UI so users should be accustomed to that.

**Comment by swhitley** [comment]
Good suggestion.  Current dropdown is similar to Workday Studio Connections page.  I'd say it's a design flaw in both places.  For now, I'm erring on the side of sticking close to Workday functionality to evolve as it evolves.  I'll continue to consider this.

![image](https://user-images.githubusercontent.com/413552/128611706-91e5cc55-cb3e-494c-899f-4bc4467ce5da.png)
